x-pack/plugins/security/server/authentication/authenticator.test.ts

```diff
@@ -198,9 +198,7 @@ describe('Authenticator', () => {
       afterEach(() => jest.resetAllMocks());
 
       it('enabled by default', () => {
-        const authenticator = new Authenticator(getMockOptions());
-        expect(authenticator.isProviderTypeEnabled('basic')).toBe(true);
```

Comment:
__note:__ we don't really need this method anymore, there is not much sense to keep it.